src/main/java/dualprotocol/sdk/sample/Creation.java

```diff
@@ -20,18 +20,18 @@
      * @param accountBody The Account body used in the creation
      * @return The newly created ANF Account
      */
-    public static CompletableFuture<NetAppAccountInner> createANFAccount(AzureNetAppFilesManagementClientImpl anfClient, String[] accountParams, NetAppAccountInner accountBody)
+    public static NetAppAccountInner createANFAccount(NetAppManagementClient anfClient, String[] accountParams, NetAppAccountInner accountBody)
     {
         try
         {
-            NetAppAccountInner anfAccount = anfClient.accounts().createOrUpdate(accountParams[0], accountParams[1], accountBody);
+            NetAppAccountInner anfAccount = anfClient.getAccounts().beginCreateOrUpdate(accountParams[0], accountParams[1], accountBody).getFinalResult();
             Utils.writeSuccessMessage("Account successfully created, resourceId: " + anfAccount.id());
 
-            return CompletableFuture.completedFuture(anfAccount);
+            return anfAccount;
         }
-        catch (CloudException e)
+        catch (Exception e)
```

Comment:
>Exception [](http://example.com/codeflow?start=15&length=9)

CloudException got replaced by AzureError (at least in Python), and should not be replaced by the broader Exception, can you please apply the equivalent for Java? Please do the same on all samples.